landscaper/pkg/gardenlet/blueprint.yaml

```
@@ -0,0 +1,109 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+jsonSchema: "https://json-schema.org/draft/2019-09/schema" # required
+imports:
+  - name: seedCluster
+    targetType: landscaper.gardener.cloud/kubernetes-cluster
+
+  - name: gardenCluster
+    targetType: landscaper.gardener.cloud/kubernetes-cluster
+
+  - name: seedBackupCredentials
+    required: false
+    schema:
+      # provider specific credentials with varying properties
```

can you add that comment as a jsonschema description (https://json-schema.org/understanding-json-schema/reference/generic.html) field?
We can then process the documentation.
.e.g.
```suggestion
      # provider specific credentials with varying properties
      description: "provider specific credentials with varying properties"
```